qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -275,9 +278,11 @@ await client.CreateCollectionAsync(
 );
 ```
 
-We recommend setting the number of shards to be a multiple of the number of nodes you are currently running in your cluster.
+The number of shards must be a multiple of the number of nodes you are currently running in your cluster.
```

Comment:
this is not a hard requirement. Some use-cases might require less shards than nodes